runtime/queries/lua/highlights.scm

```diff
@@ -0,0 +1,159 @@
+;;; Highlighting for lua
```

Comment:
what are the tradeoffs if we put these `*.scm` files in `runtime/syntax/` instead of a new `runtime/queries` location? (Other than the fact that Nvim 0.5 shipped with `queries/`)